standalone-metastore/pom.xml

```diff
@@ -80,7 +80,7 @@
     <libfb303.version>0.9.3</libfb303.version>
     <libthrift.version>0.9.3</libthrift.version>
     <log4j2.version>2.17.1</log4j2.version>
-    <mockito-all.version>1.10.19</mockito-all.version>
+    <mockito-core.version>1.10.19</mockito-core.version>
```

Comment:
why is the mockito-core.version 1.10.19 here and 2.28.2 in other places?

Reply:
The `standalone-metastore` is independent from the rest of the project. All other packages are getting `mockito-core` from the parent pom.

Initially, I backported the commit that replaced `mockito-all` with `mockito-core`. That commit wasn't changing the version. It was keeping it the same.

Hadoop is using a much newer version of `mockito-core` and due to the Hadoop code, I was forced to also update the version in the parent pom. But I wasn't getting any errors from the `standalone-metastore` module. So, I just let it be.

Now, I realize that it would be best to be consistent with the versions. I'll update it here as well and fix any issues that might come up.

Reply:
I fixed this.